### ISSUE TITLE
3.12 weight decay 图片放错了

### ISSUE THREAD

**Issue by ZhijunHou** [ShusenTang/Dive-into-DL-PyTorch#18]
最后一个图片放错啦

**Comment by ShusenTang** [comment]
> 最后一个图片放错啦

感谢提醒，已纠正
